src/javascript/grid3_javascript_qsfs_zdbs.md

```diff
@@ -0,0 +1,123 @@
+first, make sure you have your [client](./grid3_javascript_loadclient.md) prepared
+
+## Deploying ZDBs for QSFS
+
+### Example code
+
+````typescript
+import { FilterOptions, QSFSZDBSModel } from "../src";
+import { getClient } from "./client_loader";
+import { log } from "./utils";
+
+async function main() {
+    const grid3 = await getClient();
+    const qsfs_name = "zdbsQsfsDemo";
+    const qsfsQueryOptions: FilterOptions = {
+        hru: 16,
```

Comment:
if the zdbs will be distributed on 2 nodes and each zdb disk will be 1, then this should be 8 as each node will have 8 zdbs